.github/workflows/build.yml

```diff
@@ -1,20 +1,21 @@
 name: Docker builds
 
 on:
-  pull_request:
-    branches: [master]
   push:
     branches: [master]
     tags:
       - v*
+  pull_request:
+    branches: [master]
```

Comment:
Could you please put this section before `push` to get this diff out from the PR. Of course, if this reordering is not intentional - then please reorder other workflows in the new way to be consistent.

BTW, you could add the next line after the current `branches: [master]` in the `push` section to show the easy way of switching the building scope and simplify changes:
`#branches: [*]`
The same can be done for the `IS_PUSHING_IMAGES` condition.

Generally speaking, I think that we could keep `branches: [*]` if the `IS_PUSHING_IMAGES` condition by default allows pushing only in the main repo - this would allow the build execution for testing purposes and minimize needed changes to allow the pushing to uncomment 1 line and comment out the opposite one. What do you think?

Reply:
I changed the order intentionally. It has the same order as both build workflow and test workflow.

Updated as `['*']`.

Reply:
Don't you want to add commented-out alternative lines?

Reply:
Since it's already `*`, we don't need to change it if want to build images in forked repos. So I don't think we need it.

Added the comment with `IS_PUSHING_IMAGES`.